pkgs/racket-doc/scribblings/reference/pairs.scrbl

```diff
@@ -449,6 +449,53 @@ each call to @racket[proc]).
   (foldr (lambda (v l) (cons (add1 v) l)) '() '(1 2 3 4))]}
 
 
+@(define list-eval (make-base-eval))
+@examples[#:hidden #:eval list-eval
+          (require racket/list (only-in racket/function negate))
+          (require racket/list/grouping)
+          (require racket/list/iteration)]
+
+
+@; ----------------------------------------
+@section{More List Iteration}
+
+@defmodule[racket/list/iteration #:packages ("sequence-tools-lib")]
+
+The bindings in this section are provided by the @racket[sequence-tools-lib] packages 
+which acts as an extension to the base sequence libraries.
```

Comment:
It would be more consistent with other places to use `@note-lib-only[racket/list/iteration]` instead of `defmodule` and the two lines text here.

It's true that the module is provided by a different package, but it's even more relevant that the bindings aren't provided by `racket`. Also, the package name can be inferred. (I can imagine that `#:packages` might have been needed along the way, before everything was set up right.)